src/libraries/Common/src/System/Net/IPv4AddressHelper.Common.cs

```diff
@@ -217,7 +217,7 @@ internal static unsafe long ParseNonCanonical<TChar>(TChar* name, int start, ref
 
             int numberBase = IPv4AddressHelper.Decimal;
             int ch = 0;
-            long* parts = stackalloc long[3]; // One part per octet. Final octet doesn't have a terminator, so is stored in currentValue.
+            Span<long> parts = stackalloc long[3]; // One part per octet. Final octet doesn't have a terminator, so is stored in currentValue.
```

Comment:
I seem to remember this resulted in some small regressions the last time it was attempted. Can we confirm that's no longer an issue?